### ISSUE TITLE
Top Nav

### ISSUE THREAD

**Issue by lieslandr** [XavierUniversity/2016-R2X-Dev#3]
Rob to work with Jordan on main items that go in the top nav. Do we need to limit to 3? Can we add more?

**Comment by lieslandr** [comment]
Let's reset this one. We need to come up with a plan for what gets displayed, when, and set a priority.

**Comment by sparks12XU** [comment]
Since the whole thing is an include, it's possible to have sets of links based on a set of criteria as long as the class structure stays the same so any buttons after the first 3 hide at the correct break point.

**Comment by lieslandr** [comment]
I agree, but I still think we will likely have a variable order based on what is happening.
